btk/survey.py

```diff
@@ -0,0 +1,366 @@
+            atmospheric_psf_fwhm=0.074,
+            sky_brightness=22,
+            exp_time=3000,
+            zeropoint=20,
+            extinction=0,  # TODO: double-check
+        )
+    ],
+)
+
+# https://hsc-release.mtk.nao.ac.jp/doc/ deep+udeep
+
+HSC = Survey(
+    name="HSC",
+    pixel_scale=0.167,
+    effective_area=52.81,
+    mirror_diameter=8.2,
+    airmass=1.0,
+    zeropoint_airmass=1.2,
+    filters=[
+        Filter(
+            name="g",
+            atmospheric_psf_fwhm=0.72,
+            sky_brightness=21.4,
+            exp_time=600,
+            zeropoint=91.11,
+            extinction=0.13,
+        ),
+        Filter(
+            name="r",
+            atmospheric_psf_fwhm=0.67,
+            sky_brightness=20.6,
+            exp_time=600,
+            zeropoint=87.74,
+            extinction=0.11,
+        ),
+        Filter(
+            name="i",
+            atmospheric_psf_fwhm=0.56,
+            sky_brightness=19.7,
+            exp_time=1200,
+            zeropoint=69.80,
+            extinction=0.07,
+        ),
+        Filter(
+            name="y",
+            atmospheric_psf_fwhm=0.64,
+            sky_brightness=18.3,
+            exp_time=1200,
+            zeropoint=29.56,
+            extinction=0.05,
+        ),
+        Filter(
+            name="z",
+            atmospheric_psf_fwhm=0.64,
+            sky_brightness=17.9,
+            exp_time=1200,
+            zeropoint=21.53,
+            extinction=0.05,
+        ),
+    ],
+)
+
+
+# https://www.lsst.org/about/camera/features
+Rubin = Survey(
+    "LSST",
+    pixel_scale=0.2,
+    effective_area=32.4,
+    mirror_diameter=8.36,
+    airmass=1.2,
+    zeropoint_airmass=1.2,
+    filters=[
+        Filter(
+            name="y",
+            atmospheric_psf_fwhm=0.703,
+            sky_brightness=18.6,
+            exp_time=4800,
+            zeropoint=10.58,
+            extinction=0.138,
+        ),
+        Filter(
+            name="z",
+            atmospheric_psf_fwhm=0.725,
+            sky_brightness=19.6,
+            exp_time=4800,
+            zeropoint=22.68,
+            extinction=0.043,
+        ),
+        Filter(
+            name="i",
+            atmospheric_psf_fwhm=0.748,
+            sky_brightness=20.5,
+            exp_time=5520,
+            zeropoint=32.36,
+            extinction=0.07,
+        ),
+        Filter(
+            name="r",
+            atmospheric_psf_fwhm=0.781,
+            sky_brightness=21.2,
+            exp_time=520,
+            zeropoint=43.70,
+            extinction=0.10,
+        ),
+        Filter(
+            name="g",
+            atmospheric_psf_fwhm=0.814,
+            sky_brightness=22.3,
+            exp_time=2400,
+            zeropoint=50.70,
+            extinction=0.163,
+        ),
+        Filter(
+            name="u",
+            atmospheric_psf_fwhm=0.859,
+            sky_brightness=2.9,
+            exp_time=1680,
+            zeropoint=9.16,
+            extinction=0.451,
```

Comment:
The filter list being ordered, I would sort the filters by increasing wavelength => _ugrizy_

In fact you may have just swapped the name with the other values because (I might be wrong here) the atmospheric PSF size should increase from u-band to y-band and here it decreases.

Please check.